composer-json-php-ext-schema.json

```diff
@@ -40,6 +40,26 @@
                     "example": "my-extension-source",
                     "default": null
                 },
+                "os-families": {
+                    "type": "array",
+                    "minItems": 1,
+                    "description": "An array of OS families to mark as compatible with the extension. Specifying this property will mean this package is not installable with PIE on any OS family not listed here. Must not be specified alongside os-families-exclude.",
+                    "items": {
+                        "type": "string",
+                        "enum": ["Windows", "BSD", "Darwin", "Solaris", "Linux", "Unknown"],
```

Comment:
@asgrim If I understand right [your comment](https://github.com/alexandre-daubois/pie/pull/2#discussion_r1861667907), let's lowercase those ones as well?

Reply:
it actually shouldn't matter here (since we'll normalise in PIE itself anyway), but that makes sense for consistency, sure

Reply:
Alright, there you go